src/app/containers/PageHandlers/withMemo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { element } from 'prop-types';
+
+const WithMemo = Component => {
+  const ComponentMemo = React.memo(props => {
+    return <Component {...props} />;
+  });
+
+  const WithMemoContainer = props => {
+    const { history, location, match, route, ...useful } = props;
```

Comment:
The full list of props this is called with is
```
["history", "location", "match", "staticContext", "data", "service", "id", "isAmp", "pageType", "loading", "error", "bbcOrigin", "dials", "previousPath", "route"]
``` 
Seems safer to do a blacklist than a whitelist